packages/ui/src/components/va-option-list/VaOptionList.vue

```diff
@@ -150,8 +142,16 @@ export default defineComponent({
 
     const { computedError, computedErrorMessages } = useValidation(props, emit, reset, focus)
 
+    const computedProps = computed(() => ({
+      ...pick(props, ['name', 'color', 'readonly', 'leftLabel']),
+    }))
+
+    const addRef = (el: HTMLInputElement, index: number) => {
+      inputs.value[index] = el
+    }
+
     onMounted(() => {
-      if (!valueComputed.value && props.defaultValue) {
+      if (valueComputed.value === undefined && props.defaultValue) {
         selectedValue.value = props.defaultValue
```

Comment:
Looks like useStateful must be used here...

Reply:
Removed `defaultValue` prop